lib/geocode.js

```diff
@@ -53,11 +52,30 @@ function _handleArcGISResponse (error, response, body, callback) {
  * geoservice.geocode({ text: "920 SW 3rd Ave, Portland, OR 97204" }, callback);
 */
 function geocode (parameters, callback) {
-  parameters.f = parameters.f || "json";
```

Comment:
I'm good either way. Having consistency is better than not. This is one of those where everything is a grey area.